electron/ironfish/AccountManager.ts

```diff
@@ -282,10 +284,21 @@ class AccountManager
     }
   }
 
```

Comment:
Expecting this layer will not be necessary soon and we can call this client directly from React.